### PR TITLE
Rename data management zone

### PR DESCRIPTION
**Pull request by mboswell**
## Summary of the Pull Request
Rename data management zone

## PR Checklist
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "markdownlint.config": {
+        "MD028": false,
+        "MD025": {
+            "front_matter_title": ""
+        }
+    }
+}
```
Comment (marvinbuss) [inline .vscode/settings.json:8]: Can you please remove this from the files?

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please see comments below and fix merge conflicts.

File: .vscode/settings.json (hunk 1 above)
Comment (mboswell) [reply .vscode/settings.json]: @marvinbuss - Good catch - deleted file from PR.

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**